spec/controllers/msa_components_controller_spec.rb

```diff
@@ -1,10 +1,12 @@
 require 'rails_helper'
-require 'auth_test_helper'
 
 RSpec.describe MsaComponentsController, type: :controller do
   before(:each) do
-    populate_session
+    @request.env["devise.mapping"] = Devise.mappings[:user]
```

Comment:
could we extract to its own method? We're duplicating it across tests... In a similar fashion like we had the `populate_session` method... but call it something like `stub_auth` or something...

Reply:
Have created a method in `auth_support` to reduce duplication.